agent/rpc_from.go

```diff
@@ -69,6 +69,8 @@ func (a *orbAgent) handleGroupRPCFromCore(client mqtt.Client, message mqtt.Messa
 			return
 		}
 		a.handleAgentGroupRemoval(r.Payload)
+	case fleet.AgentStopRPCFunc:
+		a.Stop()
```

Comment:
this probably needs a.handlerAgentStop, where it can show the Reason. I'm not quite sure if it needs to do other things to shutdown gracefully, need to check